pcsx2/gui/FrameForGS.cpp

```diff
@@ -171,6 +182,13 @@ void GSPanel::DoResize()
 	float zoom = g_Conf->GSWindow.Zoom.ToFloat()/100.0;
 	if( zoom == 0 )//auto zoom in untill black-bars are gone (while keeping the aspect ratio).
 		zoom = std::max( (float)arr, (float)(1.0/arr) );
+	if (arr == 1 && EmuConfig.Gamefixes.FMVinSoftwareHack && g_Conf->GSWindow.IsFullscreen) {
+		//avoids crash on some systems switching HW><SW in fullscreen aspect ratio's with FMV Software switch.
+		if (zoom < 1.001)
+			zoom = (g_Conf->GSWindow.Zoom.ToFloat() + 1.0) / 100.0;
+		if (zoom > 0.999)
+			zoom = (g_Conf->GSWindow.Zoom.ToFloat() - 1.0) / 100.0;
```

Comment:
I was actually going for  +1 for zoom values for above and equal to 1 and below 1.001, and -1 for zoom values below 1 and above 0.999, but after I updated the fix I forgot to add the catch for the 1 value back to the if statements.